CHANGELOG.md

```diff
@@ -93,6 +93,7 @@ All notable changes to this project will be documented in this file.
 - use custom product versions for Hadoop, HBase, Phoenix, hbase-operator-tools, Druid, Hive and Spark ([#1173]).
 - hbase: Bump dependencies to the latest patch level for HBase `2.6.1` and `2.6.2` ([#1185]).
 - Changed default user & group IDs from 1000/1000 to 782252253/574654813 ([#1164])
+- ubi-rust-builder: Bump Rust toolchain to 1.87.0, cargo-auditable to 0.7.0 and protoc to 31.1 ([#1197]).
```

Comment:
note: This PR bumps the base image which should be included here, see [this commit](https://github.com/stackabletech/docker-images/tree/2b43530e1df74cec4b8b76232dcd1e999a0bb7a8).

note: This PR also bumps the base image of other images. Please also include those in the changelog.